docs/installation/livepeer-quickstart.md

```diff
@@ -0,0 +1,86 @@
+---
+sidebar_position: 1
+title: Livepeer Quick Start
+---
+
+Orchestrators who wish to run a node, provide transcoding services, or develop video streaming to integrate with your applications can quickly get up and running in just a few steps:
+
+1. Check the specified requirements and pre-requisites  
+2. Follow the Installation workflow for 
+> - [video miners](/livepeer-quick-start/#start-video-mining)
+> - [developers](/livepeer-quick-start/#start-developing)
+
+>**Recommendations** If you are new to Livepeer, you can familiarize yourself futher about our platform capabilities highlighted in [Core Concepts](/core-concepts/core-concepts.md). 
+
+### Technical Requirements
+
+At Livepeer we support:
```

Comment:
how about:
"`go-livepeer` supports:"?

two reasons - first, we want to be clear that go-livepeer is just one potential implementation; second, I want to move away from implying that the company controls everything